docs/reference/api-conventions.asciidoc

```diff
@@ -30,27 +38,32 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
 
 The defaults settings for the above parameters depend on the API being used.
 
-Some multi index APIs also support the following url query string parameter:
+Some multi-target APIs that can target indices also support the following query
+string parameter:
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=ignore_throttled]
 
-NOTE: Single index APIs such as the <<docs>> and the
-<<indices-aliases,single-index `alias` APIs>> do not support multiple indices.
+NOTE: Single index APIs, such as the <<docs>> and the
+<<indices-aliases,single-index `alias` APIs>>, do not support multi-target
+syntax.
 
 [[date-math-index-names]]
-=== Date math support in index names
+=== Date math support in target names
+
+Most APIs that accept a `<data-stream>`, `<index>`, or `<target>` request path
+parameter support _date math_.
```

Comment:
Just a heads-up, but data stream names don't support date math since a single data stream is supposed to handle a potentially infinite stream of data and the data stream's backing indices are named according to their generation and not the date on which they were created.

Reply:
Thanks @danhermann. I've removed my changes to this section/page.

Technically, you can still search a data stream name using date math. For example, `GET /%3Clogs-%7Bnow%2Fd%7D%3E/_search` would search the `logs-2020.06.30` data stream.

But first users would need to create a data stream with a date component , which we probably shouldn't encourage. It seems like a corner case we'd be better off not mentioning.